src/contexts/Localisation.tsx

```diff
@@ -4,21 +4,25 @@ import strings from '../utils/defaultLanguageStrings'
 import { getRequest } from '../utils/helpers/fetchMethods'
 import { mapValues } from 'lodash'
 
-const { defaultLanguageCode, pluginsFolder } = config
+const { pluginsFolder } = config
 
-const initialLanguageCode = localStorage.getItem('language') ?? defaultLanguageCode
+const savedLanguageCode = localStorage.getItem('language')
```

Comment:
Hmm are we always storing on localStorage? I thought this would be included in the user details... So after changing the selected language this would be persisted in the user created or edited in an existing user.

Not sure about implementation wise, do you think this would be much more difficult to do?

Reply:
Do you mean saving it in the database on the user table?

I thought about that, then I realised -- that needs them to be logged in before it can be determined. So they're still going to have to use local storage or manually select their language on the Login page. And after that we have it anyway, so no need to save to database!

You get what I mean?